bundles/org.eclipse.e4.ui.model.workbench/src/org/eclipse/e4/ui/model/application/ui/menu/util/MenuAdapterFactory.java

```diff
@@ -104,7 +104,7 @@ public boolean isFactoryForType(Object object) {
 	 * <!-- end-user-doc -->
 	 * @generated
 	 */
-	protected MenuSwitch<Adapter> modelSwitch = new MenuSwitch<Adapter>() {
+	protected MenuSwitch<Adapter> modelSwitch = new MenuSwitch<>() {
```

Comment:
This is generated code so shouldn't be manually edited. @merks should there be some update in EMF itself or regeneration should help?

Reply:
The generator uses the configured cleanup actions which would make this change, but I it doesn't apply such an action if the generated file is unchanged by generation.  So better would be to do a manual cleanup of the entire project and then regenerate the model(s).